Sources/PackageModel/UserToolchain.swift

```diff
@@ -522,6 +525,18 @@ public final class UserToolchain: Toolchain {
         self.isSwiftDevelopmentToolchain = false
         #endif
 
+        if let customInstalledSwiftPMConfiguration = customInstalledSwiftPMConfiguration {
```

Comment:
This can use shorthand `if let`

```suggestion
        if let customInstalledSwiftPMConfiguration {
```